packages/driver/src/cy/commands/querying/querying.ts

```diff
@@ -148,375 +150,8 @@ function getAlias (selector, log, cy) {
   }
 }
 
-interface InternalGetOptions extends Partial<Cypress.Loggable & Cypress.Timeoutable & Cypress.Withinable & Cypress.Shadow> {
-  _log?: Log
-  _retries?: number
-  filter?: any
-  onRetry?: Function
-  verify?: boolean
-}
-
-interface InternalContainsOptions extends Partial<Cypress.Loggable & Cypress.Timeoutable & Cypress.CaseMatchable & Cypress.Shadow> {
-  _log?: Log
-}
-
 export default (Commands, Cypress, cy, state) => {
-  /*
```

Comment:
In this PR, I've resolved the temporary and confusing state of having two implementations of `.get()`. It was ugly, but now it's gone!